trillian/trampoline/trampoline.go

```diff
@@ -36,7 +37,13 @@ func main() {
 		}
 	}
 
-	fmt.Println(args)
-
-	syscall.Exec(*target, args, os.Environ())
+	for {
```

Comment:
`docker-compose` has no sense of "wait until a container is ready", you have to implement it yourself and this was quickest. Obviously you'd want something more robust for production.